markdown/1-Introduction/exercises/ex_10/index.md

```diff
@@ -5,63 +5,11 @@ permalink: /intro-exercises/ex_10/
 breadcrumb: 1-Introduction
 breadcrumb2: ex_10
 breadcrumb3: 1introduction
+breadcrumb4: ch1ex10
```

Comment:
Why did you add new breadcrumbs? I do not recall discussing any such thing in the meetings.

Reply:
Sent a new PR by removing the unnecessary Breadcrumbs.